### PR TITLE
Cleanup deprecated

### PR DESCRIPTION
**Pull request by leondavis1**
First pass at removing modules that aren't being used; also replaces usage of ptsa.data.common.PropertiedObject with Enthought Traits

### CODE REVIEW

File: ptsa/data/__init__.py [hunk 1]
```diff
@@ -1,7 +0,0 @@
-# FIXME: all of this is old PTSA stuff that should be removed
```
Comment (mivade) [inline ptsa/data/__init__.py]: This file should still exist even if empty.

File: ptsa/data/filters/DataChopper.py [hunk 2]
```diff
@@ -3,45 +3,50 @@
...
 
-class DataChopper(PropertiedObject,BaseFilter):
+class DataChopper(traits.api.HasTraits):
     """
     EventDataChopper converts continuous time series of entire session into chunks based on the events specification
     In other words you may read entire eeg session first and then using EventDataChopper
     divide it into chunks corresponding to events of your choice
     """
-    _descriptors = [
+    start_time = traits.api.Float
```
Comment (mivade) [inline ptsa/data/filters/DataChopper.py]: Should we use `CFloat` instead of `Float` everywhere?

File: ptsa/data/__init__.py (hunk 1 above)
Comment (leondavis1) [reply ptsa/data/__init__.py]: This file does still exist, but Github doesn't format the diff differently for empty files vs removed files

**Comment by leondavis1** [comment]
Addresses #140

**Pull request merged**